tests/scripts/common_functions.sh

```diff
@@ -174,38 +174,48 @@ function deploy_autotune() {
 	${cmd}
 	
 	status="$?"
-	# Check if autotune is deployed 
-	if [ "${status}" -eq "1" ]; then
+	# Check if autotune is deployed.
+	# Ignore the status check if ignore_deployment_status_check is set to "1".
+	# In case of configmap yaml tests we need not check if autotune has deployed properly during the setup since it is done as part of the test.
+	if [[ "${status}" -eq "1" && "${ignore_deployment_status_check}" != "1" ]]; then
 		echo "Error deploying autotune" >>/dev/stderr
-		echo "See ${PWD}/tests/setup.log for more info" >>/dev/stderr
+		echo "See ${AUTOTUNE_SETUP_LOG}" >>/dev/stderr
 		exit -1
 	fi
 }
 
+# Remove the prometheus setup
+# output: Remove all the prometheus dependencies
+function prometheus_cleanup() {
+	kubectl_cmd="kubectl"
+	prometheus_pod_running=$(${kubectl_cmd} get pods --all-namespaces | grep "prometheus-k8s-1"| awk '{print $4}')
+	if [ "${prometheus_pod_running}" == "Running" ]; then
+		./scripts/prometheus_on_minikube.sh -t
+	fi
+}
+
 # Remove the autotune setup
 # output: Remove all the autotune dependencies
 function autotune_cleanup() {
+	RESULTS_LOG=$1
+	
+	# If autotune cleanup is invoke through -t option then setup.log will inside the given result directory
+	if [ ! -z "${result_dir}" ]; then
+		AUTOTUNE_SETUP_LOG="${RESULTS_LOG}/autotune_setup.log"
+		AUTOTUNE_REPO="${AUTOTUNE_REPO}/autotune"
```

Comment:
Isn't autotune repo already initialized on top?

Reply:
when we call cleanup using -t option current directory will be in tests  so autotune_repo at the top will get initialized to 2 directory before.
```
CURRENT_DIR="$(dirname "$(realpath "$0")")"
AUTOTUNE_REPO="${CURRENT_DIR}/../.."
```